cms/serializers_test.py

```diff
@@ -225,3 +230,65 @@ def test_serialize_course_page_with_flex_price_form_as_child_no_program(
             "effort": course_page.effort,
         },
     )
+
+
+@pytest.mark.parametrize(
+    "own_program_has_form,related_program,related_program_has_form",
+    [
+        [True, False, False],
+        [True, True, False],
+        [False, True, False],
+        [False, True, True],
+        [True, True, True],
+    ],
+)
+def test_serialized_course_finaid_form_url(
+    own_program_has_form, related_program, related_program_has_form
+):
+    """
+    Tests the financial assistance form URL returned for a course that's in a
+    program that has no financial assistance form and is related to a program
+    that does have a financial assistance form. (The course should get the
+    form URL for the program that does have the financial assistance form.)
```

Comment:
I would just update this docstring to better describe the different cases defined by your parameter values (`own_program_has_form,related_program,related_program_has_form`).  This unit test is testing more situations than the current docstring describes.

Reply:
Got this in https://github.com/mitodl/mitxonline/pull/1940/commits/0531caa2c5967acc0cdafbcf593ac8f7882125c0 - since this is just a comment change, going ahead with merging.